build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -69,12 +69,48 @@ export interface NavigationBarIOSProps {
      * The color of the title view
      */
     titleColor?: string;
+
+    /**
+     * The logo
+     */
+    logo?: ImageURISource;
+    /**
+     * The menu overflow image
+     */
+    overflowImage?: ImageURISource;
+    /**
+     * The navigation button image
+     */
+    navigationImage?: ImageURISource;
+    /**
+     * Handles navigation button click events
+     */
+    onNavigationPress?: () => void
+    /**
+     * Determines when this item should appear in the navigation bar
+     */
+    show?: 'ifRoom' | 'never' | 'always';
```

Comment:
The `show` prop should be on the `BarButtonProps`. Don't forget to copy the changes into the other [navigation-react-native.d.ts file](https://github.com/grahammendick/navigation/blob/master/NavigationReactNative/test/node_modules/%40types/navigation-react-native.d.ts)

Reply:
The `show` prop should be on the `BarButtonProps`not the `NavigationBarProps`